src/components/RenderConfig/ColormapConfigComponent/ColormapConfigComponent.tsx

```diff
@@ -63,7 +63,7 @@ export class ColormapConfigComponent extends React.Component<ColormapConfigProps
                     </FormGroup>
                 )}
                 <FormGroup inline={true}>
-                    <Button minimal={true} rightIcon={this.extendBiasContrast ? "double-chevron-up" : "double-chevron-down"} alignText={"right"} small={true} onClick={this.switchExtendBiasContrast}>
+                    <Button minimal={true} className={"bp3-bias-contrast"} rightIcon={this.extendBiasContrast ? "double-chevron-up" : "double-chevron-down"} alignText={"right"} small={true} onClick={this.switchExtendBiasContrast}>
```

Comment:
please rename it to `bias-contrast-button`

Reply:
Fixed!